scripts/make-league.js

```diff
@@ -0,0 +1,153 @@
+const { League } = require('../models/league.js');
+
+const readline = require('readline').createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+const exec = require('child_process').execSync;
+const path = require('path');
+const os = require('os');
+const fs = require('fs').promises;
+
+async function query(message) {
+    return new Promise((resolve, _) => {
+        readline.question(message + ': ', (answer) => {
+            resolve(answer);
+        });
+    });
+}
+
+async function okPrompt() {
+    const response = await query("Is this ok? [y/n]");
+    switch (response) {
+        case 'y':
+        case 'Y':
+            return true;
+        case 'n':
+        case 'N':
+            return false;
+        default:
+            console.log(`I don't understand, try again`);
```

Comment:
Pissmonger would suffer no fools while taking input, please fix this politeness.

Reply:
Fixed